clients/web/src/init.js

```diff
@@ -168,7 +168,7 @@ _.extend(girder, {
         }).then(function () {
             girder.currentUser = null;
 
-            girder.events.trigger('g:login', null);
+            girder.events.trigger('g:logout');
```

Comment:
@mgrauer this change would break backward compatibility; the `g:login` event is triggered on both login and logout (it's poorly named). That's by design so that pages that need to update when the user changes only have to bind to one event rather than two.